ci-operator/config/stolostron/search-indexer/stolostron-search-indexer-main.yaml

```diff
@@ -0,0 +1,79 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  open-cluster-management_builder_go1.17-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.17-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    open-cluster-management_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:go1.17-linux
+  to: search-indexer
+promotion:
+  name: "2.5"
+  namespace: stolostron
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-indexer
+    env:
+      IMAGE_REPO: search-indexer
+      REGISTRY_ORG: stolostron
+    workflow: ocm-ci-image-mirror
+- as: backplane-fast-forward
+  postsubmit: true
+  steps:
+    env:
+      DESTINATION_BRANCH: release-2.5
```

Comment:
If this is backplane, it's going into `backplane-2.0`.  If not... then all the backplane references are wrong.

Reply:
this is not going into backplane, it's meant to go into ACM, see above comment